src/components/dismissible-box/dismissible-box.pw.tsx

```diff
@@ -0,0 +1,179 @@
+        );
+
+        await expect(dismissibleBoxDataComponent(page)).toHaveCSS(
+          "border-left-color",
+          color
+        );
+      });
+    }
+  );
+
+  [150, 350].forEach((width) => {
+    test(`should apply correct width to component when width prop set to ${width}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DefaultDismissibleBox width={`${width}px`} />);
+
+      await expect(dismissibleBoxDataComponent(page)).toHaveAttribute(
+        "width",
+        `${width}px`
+      );
+
+      await assertCssValueIsApproximately(
+        dismissibleBoxDataComponent(page),
+        "width",
+        width
+      );
+    });
+  });
+
+  ([
+    ["light", "rgb(255, 255, 255)"],
+    ["dark", "rgb(230, 235, 237)"],
+  ] as [DismissibleBoxProps["variant"], string][]).forEach(
+    ([variant, color]) => {
+      test(`should apply correct styling to component with variant prop set to ${variant}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<DefaultDismissibleBox variant={variant} />);
+
+        await expect(dismissibleBoxDataComponent(page)).toHaveCSS(
+          "background-color",
+          color
+        );
+      });
+    }
+  );
+
+  ([
+    [undefined, "8px"],
+    ["borderRadius000", "0px"],
+    ["borderRadius025", "2px"],
+    ["borderRadius050", "4px"],
+    ["borderRadius200", "16px"],
+    ["borderRadius400", "32px"],
+  ] as [DismissibleBoxProps["borderRadius"], string][]).forEach(
+    ([borderRadius, expected]) => {
+      test(`applies the expected border radius when ${borderRadius} passed to borderRadius prop`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<DefaultDismissibleBox borderRadius={borderRadius} />);
+        await expect(dismissibleBoxDataComponent(page)).toHaveCSS(
+          "border-radius",
+          expected
+        );
+      });
+    }
+  );
+});
+
+test.describe("Check events", () => {
+  test("should call onClose callback when a mouse click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    await mount(
+      <DefaultDismissibleBox
+        onClose={() => {
+          callbackCount += 1;
+        }}
+      />
+    );
+
+    await icon(page).click();
+
+    await expect(callbackCount).toEqual(1);
+  });
+
+  ["Space", "Enter"].forEach((key) => {
+    test(`should call onClose callback when a keyboard key ${key} event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+      await mount(
+        <DefaultDismissibleBox
+          onClose={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      await page.keyboard.press("Tab");
+      await page.keyboard.down(key);
+      await expect(callbackCount).toEqual(1);
+    });
+  });
+});
+
+test.describe("Accessibility tests", () => {
+  test("should pass accessibility tests for DismissibleBox DefaultDarkVariant story", async ({
```

Comment:
**suggestion**: as we're tying to clean up test descriptions, perhaps we don't need to reference the component name in each individual test, instead include it in the describe above. That way we're not repeating the component name every time